### PR TITLE
Removes credits from the repo

### PR DESCRIPTION
**Pull request by Cyberboss**
Fucking licensing memes for the avatars. Now they go in the config folder. This closes #30497 because it effectively disables them for now

### CODE REVIEW

**Comment by Xhuis** [comment]
Uh... works for me!

**Comment by Jordie0608** [comment]
Unless WIP I don't see how this is the same as #30497

**Comment by Cheridan** [comment]
Thanks!

**Comment by YakumoChen** [comment]
credits are literally in the changelog, what even is the point

I'm in the credits but I don't even care. they take up space on the game screen when I'd prefer to post-round meme and the verb is hidden and most of the time I don't even know it exists so I can't turn it off.

does this PR add a pref like the last PR because I'd rather not have to see them at all, if anyone wants to complain they can just point at the changelog which is literally the same thing anyways but doesn't shove it in your face every round.

Before someone interjects about 'every movie has credits', you don't have to watch those either. You can just turn it off when the credits start rolling. In the game you have to actually wait for them to finish before you can keep playing.

**Comment by Bawhoppen** [comment]
You don't have to watch them. You can always disconnect our walk away from your computer just like in an actual movie.

**Comment by jughu** [comment]
Im in the credits for shitpostibg

**Comment by jughu** [comment]
Also this is just your case of REEE MY END GAME GRIEF

**Comment by Iamgoofball** [comment]
@Jordie0608 There's problems with licensing on avatars, my avatar is technically copywritten

Moving it to a "config" means if we make the server tools generate it on git update for the server, credits are always up to date for both the server and the downstreams.

**Comment by Jordie0608** [comment]
Ok sure, but that's no answer to why it's said to be the same as the player preference for credits of #30497 that has me wondering if this is meant to be a WIP.

**Comment by Iamgoofball** [comment]
He doesn't, re-read the OP. It's confusing, yeah, but it's not that.

**Comment by Cyberboss** [comment]
No point in adding the preferences PR for now as there's no way to effectively the credits other than server ops doing it themselves so it's effectively disabled until I come up with something. I'll take that time to rework them and then if people still hate it it can be reopened.

**Comment by Cyberboss** [comment]
Also someone explain this travis error

**Comment by Iamgoofball** [comment]
I told you make the fucking server tools scrape git history when updating the server and recompile the dmi

**Comment by AnturK** [comment]
I don't really see how this addresses the other PR, but i'll take you on your word it's going to get improved later.

**Comment by Arianya** [comment]
It seems weird to reject/close a pref insertion on the basis that the feature is being disabled *for now* but will return at a later date.

**Comment by lzimann** [comment]
@Cyberboss

**Comment by Cyberboss** [comment]
I literally don't understand the error

**Comment by nicbn** [comment]
```
code/_onclick/hud/credits.dm:15:error: credits_icon: invalid variable
```

File: code/_onclick/hud/credits.dm [hunk 1]
```diff
@@ -5,17 +5,22 @@
 
 /client/proc/RollCredits()
 	set waitfor = FALSE
+	var/path = "[GLOB.config_dir]credits.dmi"
+	if(!fexists(path))
+		return
+	var/icon/credits_icon = new(path)
```
Comment (Cyberboss) [inline code/_onclick/hud/credits.dm]: variable defined here

File: code/_onclick/hud/credits.dm [hunk 2]
```diff
@@ -5,17 +5,22 @@
...
 	set waitfor = FALSE
+	var/path = "[GLOB.config_dir]credits.dmi"
+	if(!fexists(path))
+		return
+	var/icon/credits_icon = new(path)
 	LAZYINITLIST(credits)
 	var/list/_credits = credits
 	verbs += /client/proc/ClearCredits
-	var/static/list/credit_order_for_this_round = list("Thanks for playing!") + (shuffle(icon_states('icons/credits.dmi')) - "Thanks for playing!")
+	var/static/list/credit_order_for_this_round = list("Thanks for playing!") + (shuffle(icon_states(credits_icon)) - "Thanks for playing!")
```
Comment (Cyberboss) [inline code/_onclick/hud/credits.dm]: And used here, I don't get it

Comment (Iamgoofball) [reply code/_onclick/hud/credits.dm]: Static maybe?

**Comment by KorPhaeron** [comment]
Need to figure out a solution for updating the dmi now though

**Comment by Iamgoofball** [comment]
SERVER

TOOLS

**Comment by Cyberboss** [comment]
In 2020 with TGS4 we actually have the technology for this now

**Pull request merged**